### PR TITLE
Delete looks same adapter #61

### PR DESCRIPTION
**Pull request by adela-istrate-hs**

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs @flavius-tirnacop-hs @valentin-radulescu-hs Can you review?

File: index.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
 
 Mugshot.adapters = {
-  WebdriverIO: require('./lib/adapters/webdriverio.js'),
-  LooksSame: require('./lib/adapters/looks-same.js')
+  WebdriverIO: require('./lib/adapters/webdriverio.js')
 };
 
 Mugshot.interfaces = {
   browser: require('./lib/interfaces/browser.js'),
-  differ: require('./lib/interfaces/differ.js'),
```
Comment (andrei-picus-hs) [inline index.js]: Don't remove the interface. They're needed so others can build adapters.

File: index.js [hunk 2]
```diff
@@ -1,13 +1,11 @@
 var Mugshot = require('./lib/mugshot.js');
 
 Mugshot.adapters = {
-  WebdriverIO: require('./lib/adapters/webdriverio.js'),
-  LooksSame: require('./lib/adapters/looks-same.js')
+  WebdriverIO: require('./lib/adapters/webdriverio.js')
```
Comment (andrei-picus-hs) [inline index.js]: We should remove the WebdriverIO adapter as well and use the package instead.

Comment (andrei-picus-hs) [inline index.js]: Saw that it's being done in #60. You should update the PR name to indicate that we're only removing the differ adapter.

**Comment by flavius-tirnacop-hs** [comment]
LGTM

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs Can I merge this?

File: package.json [hunk 3]
```diff
@@ -10,14 +10,16 @@
...
+    "png-crop": "0.0.0",
+    "mugshot-looks-same": "^1.0.0"
   },
   "devDependencies": {
     "chai": "^3.2.0",
     "eslint": "3.0.1",
     "eslint-plugin-ubervu": "0.1.0",
     "istanbul": "^0.3.17",
     "mocha": "^2.2.5",
+    "mugshot-looks-same": "^1.0.0",
```
Comment (andrei-picus-hs) [inline package.json]: A dependency shouldn't be both for prod and dev. Remove the dev one.

**Comment by andrei-picus-hs** [comment]
Merge after addressing that comment.

**Pull request merged**